ac1/_sources/regression_and_line_of_best_fit/outliers.rst

```diff
@@ -88,3 +98,5 @@ important considerations to keep in mind.
    temperature, but you can say that there is a strong relationship between
    latitude and temperature, and that greater latitudes are associated with
    lower January temperatures.
+   
+.. _outliers.: https://github.com/google/applied-computing-series/blob/master/ac1/_sources/basic_descriptive_statistics/outliers_and_skew.rst#id2
```

Comment:
Can't we use an internal link here? As we have been doing for the rest of the book, using `:ref:`?